jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -1275,4 +1276,16 @@ public static DslCsvDataSet csvDataSet(String csvFile) {
     return new DslCsvDataSet(csvFile);
   }
 
+
+  /**
+   * Builds a DslWeightedSwitchController which wraps logic of BlazeMeter WeightedSwitchController.
```

Comment:
consider adding a description that might be more descriptive for users that are not familiar with WeightedSwitchController blazemeter plugin.